### PR TITLE
Queue consumer tz

### PR DESCRIPTION
**Pull request by mponomar**
Fixes timezone for datetime values in queue consumers.  Also pushes queue consumer code into its own plugin.

### CODE REVIEW

File: db/translistener.c [hunk 1]
```diff
@@ -630,6 +632,7 @@ static void append_field(struct byte_buffer *bytes, struct field *f,
 
             outopts.flags = FLD_CONV_TZONE;
             strcpy(outopts.tzname, record->trans->iq->tzname);
+            strcpy(outopts.tzname, "America/New_York");
```
Comment (akshatsikarwar) [inline db/translistener.c]: ??

File: db/process_message.c [hunk 2]
```diff
@@ -5112,9 +4802,19 @@ int process_command(struct dbenv *dbenv, char *line, int lline, int st)
         rc = handle_lrl_tunable(tok, ltok, line + st, lline - st, DYNAMIC);
         switch (rc) {
         case TUNABLE_ERR_OK: break;
-        case TUNABLE_ERR_INVALID_TUNABLE:
-            logmsg(LOGMSG_ERROR, "Unknown command <%.*s>\n", ltok, tok);
+        case TUNABLE_ERR_INVALID_TUNABLE: {
+            /* One more chance - this could be handled by a plugin */
+            struct message_handler *h;
+            LISTC_FOR_EACH(&dbenv->message_handlers, h, lnk) {
```
[the remark this answers is not in the document]
Comment (mponomar) [reply db/process_message.c]: 'queue stat' and friends, or whatever custom handlers a module may want to install

**Pull request merged**